src/amitaRJun20/PrintMiddleCharOfWord.java

```diff
@@ -0,0 +1,24 @@
+package amitaRJun20;
+
+public class PrintMiddleCharOfWord {
+	
+	void findMiddleChar() {
+		String input="Harsh Maulik";
+		System.out.println("print middle character of : "+input);
+		String[] array=input.split(" ");
+		for(int index=0;index<array.length;index++) {
+			String name=array[index];
+			int middleChar=(name.length())/2;
+			if(name.length()%2==0) 
```

Comment:
without condition also we can do it.
Think about it.
Harsh --> 2
Maulik --> 2

what if we say name.charAt(name.length()-1)